manager/install_cortex.sh

```diff
@@ -96,11 +96,19 @@ function setup_istio() {
     sleep 5
   done
 
-  echo -n "."
```

Comment:
nit: I think we should have `echo -n "."` on line 98 (before `helm template`), and then we can remove the one that's two lines below it.